### PR TITLE
AMB-07 Create delivery employee project link table

### PR DESCRIPTION
**Pull request by james-liam-butler**

### CODE REVIEW

**Review by CharlotteMKainos — COMMENTED** [review]
Add primary key

**Review by dana-kainos — CHANGES_REQUESTED** [review]
startDate and finishDate are not required in this project

**Review by dana-kainos — APPROVED** [review]

File: src/main/resources/db/migration/V4__create_del_emp_project_link_table.sql [hunk 1]
```diff
@@ -0,0 +1,7 @@
+CREATE TABLE projectDeliveryEmployees (
```
Comment (lukemKainos) [inline src/main/resources/db/migration/V4__create_del_emp_project_link_table.sql]: whats the naming convention for the file?

File: src/main/resources/db/migration/V4__create_del_emp_project_link_table.sql [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+    projectId INT NOT NULL,
+    deliveryEmployeeId INT NOT NULL,
+    FOREIGN KEY(projectId) REFERENCES project(projectId),
+    FOREIGN KEY(deliveryEmployeeId) REFERENCES deliveryEmployee(deliveryEmployeeId),
+    PRIMARY KEY(projectId, deliveryEmployeeId)
```
Comment (lukemKainos) [inline src/main/resources/db/migration/V4__create_del_emp_project_link_table.sql]: is this the right order of primary and foreign key

**Review by lukemKainos — CHANGES_REQUESTED** [review]

File: src/main/resources/db/migration/V4__create_del_emp_project_link_table.sql (hunk 1 above)
Comment (james-liam-butler) [reply src/main/resources/db/migration/V4__create_del_emp_project_link_table.sql]: changed

File: src/main/resources/db/migration/V4__create_del_emp_project_link_table.sql (hunk 2 above)
Comment (james-liam-butler) [reply src/main/resources/db/migration/V4__create_del_emp_project_link_table.sql]: changed

**Review by lukemKainos — APPROVED** [review]

**Pull request merged**